policy-engine/src/graph.rs

```diff
@@ -84,8 +85,3 @@ pub(crate) fn index(req: HttpRequest<State>) -> Box<Future<Item = HttpResponse,
         }
     }
 }
-
-#[derive(Clone)]
-pub struct State {
```

Comment:
I would personally leave this here, instead of moving it up to `main.rs` (and derive `Debug`, while at it).

Reply:
The reason I moved it out is that it's no longer specific to `graph.rs`, as `openapi.rs` also makes use of it, so there would be a dependency on the former by the latter, which I wanted to avoid.

Reply:
Ok, let's move it then.